packages/types/src/availabilitySlot.ts

```diff
@@ -40,3 +46,38 @@ export type SubSlot = {
 };
 
 export type GeneralSlot = Slot | SubSlot;
+
+// API Payloads / Queries
+export type FindAvailabilitySlotsApiParams = {
+  userId: number;
+  after: string;
+  before: string;
+  pagination: SkippablePagination;
+};
+
+// API Requests
+export type SetAvailabilitySlotsApiRequest = {
```

Comment:
We use the term `payload`. There is nothing especial about it other than **constancy**.

```suggestion
export type SetAvailabilitySlotsApiPayload = {
```